src/darwin/Framework/CHIP/templates/MTRClusterNames.zapt

```diff
@@ -0,0 +1,8 @@
+{{> header excludeZapComment=true}}
+
+#import <Matter/MTRClusterConstants.h>
+#import <Matter/MTRDefines.h>
+#import <stdint.h>
+
+MTR_EXPORT MTR_NEWLY_AVAILABLE NSString * MTRClusterNameForID(MTRClusterIDType clusterID);
+MTR_EXPORT MTR_NEWLY_AVAILABLE NSString * MTRAttributeNameForID(MTRClusterIDType clusterID, MTRAttributeIDType attributeID);
```

Comment:
Should have API documentation that describes what's returned for unknown IDs.

Reply:
Done.